charts/kyverno/templates/hooks/clusterrole.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "kyverno.pre-upgrade-job.name" . }}
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-weight": "-10"
+rules:
+- apiGroups: [""]
+  resources: [configmaps]
+  verbs: [get, list, patch, create, delete]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: [customresourcedefinitions]
+  verbs: [get, list, patch, create, delete, watch]
+- apiGroups: ["apps"]
+  resources: [deployments]
+  verbs: [get, list, patch, create, delete]
```

Comment:
I'm not seeing any kubectl references to deployments, so this set of capabilities shouldn't be needed.